packages/vite/types/chokidar.d.ts

```diff
@@ -117,7 +117,7 @@ export interface WatchOptions {
    * (the path), second time with two arguments (the path and the
    * [`fs.Stats`](https://nodejs.org/api/fs.html#fs_class_fs_stats) object of that path).
    */
-  ignored?: any
+  ignored?: string[]
```

Comment:
Looks like `ignored` can also be of type `string` when only one entry is needed, or [a regex](https://github.com/paulmillr/chokidar#api). I don't find proper docs for it, maybe it is better to first send this change to chokidar so it is properly discussed there, and then we can port it here?

Reply:
Yes, you are right. But it seems to introduce another problem. we make the assumption that it is just an array.
https://github.com/vitejs/vite/blob/eef51cb37db98a1ad9a541bdd3cd74736ff8488d/packages/vite/src/node/server/index.ts#L320-L322
I will close this and open another PR to fix it. Thanks for all the guidance.